tasks/section3.yml

```diff
@@ -451,7 +553,9 @@
       - rule_3.6.1
 
 - name: "SCORED | 3.6.1 | PATCH | Ensure iptables is installed"
-  command: /bin/true
+  yum:
+      name=iptables
+      state=installed
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md